### PR TITLE
fix error in logits process in hpu graph

### PR DESCRIPTION
**Pull request by sywangyi**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

**Comment by sywangyi** [comment]
I use the bloom-560m for test. the infer request is
 curl 127.0.0.1:8080/generate \
     -X POST \
     -d '{"inputs":"What is Deep Learning??","parameters":{"max_new_tokens":64, "do_sample": false, "temperature":0.9}}' \
     -H 'Content-Type: application/json' 

w/o the fix the generated_text is 
{"generated_text":" Well you think, is it? I think it is. This is a very simple way of learning machines, and for me it is one of the best learning techniques I have ever seen. This new work in my handbook is part of the computer vision training course, I believe this course and this book provide a good"}
{"generated_text":" lot gl good very lot lot great good great solid good lot much very lot great clear basic method very guide great great very foundation a comprehensive lot good useful complete very better way lot well very great good great different useful unique lot great lot wonderful great lot a brief basic great very very great complete good good great quick very lot great"}

{"generated_text":" very solid good lot good great great lot very good lot very complete basic wonderful good complete guide great wonderful very good very really great better good great useful very guide good better good starting good great high quick lot lot very very basic much very deeper well lot great lot really one lot good lot better better much solid lot foundation lot solution"}

perf is 35msec/token

with the fix, output is
 {"generated_text":" I just love to learn from what I can get my hands on. I love my teachers, I have so much to learn from.\nI think classrooms become a huge classroom. There are lots of work it takes. And there are lots of lots of little things to-does that I can do to improve. I"}
{"generated_text":" And the one thing that I can't understand...How can it be anything but mathematical?\"The fact that is all a question in the past is one reason why I love all my life and this day in this day and time. One of those thing in the past the way my life can change by our mind and"}
 {"generated_text":" As one of the most famous Deep Learning algorithms (which started on the work in Artificial Intelligence), DNN NN is a very powerful language for machine learning and it is based upon using deep structures. It used using the knowledge of the human brain which is an extremely powerful language. By studying DNNNNNN, we can"}

perf is 21msec/token

**Review by regisss — COMMENTED** [review]
Can you explain a bit more why we use HPU graphs only after the 1st token?

Also, I guess this solves #383 right?

**Comment by sywangyi** [comment]
> Can you explain a bit more why we use HPU graphs only after the 1st token?
> 
> Also, I guess this solves #383 right?

the shape of score for 1st token is different from >1st token. also, I think what I see is same with https://github.com/huggingface/optimum-habana/issues/383, and it should fix the issue

**Review by regisss — APPROVED** [review]
